cs/mailing.texy

```diff
@@ -179,7 +192,24 @@ $mailer->send($mail);
 \--
 
 
-Můžeme si také vytvořit vlastní mailer, jde o objekt implementující rozhraní [Nette\Mail\IMailer |api:].
+Fallback Mailer
+---------------
+
+Máme-li občas potíže s odesíláním emailů (např. výpadek smtp serveru), můžeme využít `FallbackMailer`, který umí v případě chyby automaticky zopakovat pokus o odeslání, buď pomocí stejného nebo pomocí alternativního maileru.
```

Comment:
Dá se věta ukončit slovy „, jakožto další parametry konstruktoru“ a není potřeba na to dělat příklad.